`repo_and_version_of_pull_request_body` crashes if there is a PR with empty body

`repo_and_version_of_pull_request_body` crashes if there is a PR with empty body.

```
ERROR: MethodError: no method matching occursin(::String, ::Nothing)
Closest candidates are:
  occursin(::Union{AbstractChar, AbstractString}, ::AbstractString) at strings/search.jl:628
  occursin(::Any) at strings/search.jl:642
Stacktrace:
 [1] repo_and_version_of_pull_request_body(body::Nothing)
   @ RegistryCI.TagBot ~/.julia/packages/RegistryCI/3KBMb/src/TagBot/TagBot.jl:48
 [2] #1
   @ ~/.julia/packages/RegistryCI/3KBMb/src/TagBot/cron.jl:5 [inlined]
 [3] iterate
   @ ./generator.jl:47 [inlined]
 [4] _collect(c::Vector{GitHub.PullRequest}, itr::Base.Generator{Vector{GitHub.PullRequest}, RegistryCI.TagBot.var"#1#3"}, #unused#::Base.EltypeUnknown, isz::Base.HasShape{1})
   @ Base ./array.jl:807
 [5] collect_similar
   @ ./array.jl:716 [inlined]
 [6] map
   @ ./abstractarray.jl:2933 [inlined]
 [7] handle_cron(event::Dict{String, Any})
   @ RegistryCI.TagBot ~/.julia/packages/RegistryCI/3KBMb/src/TagBot/cron.jl:5
 [8] main()
   @ RegistryCI.TagBot ~/.julia/packages/RegistryCI/3KBMb/src/TagBot/TagBot.jl:43
 [9] top-level scope
   @ none:1
Error: Process completed with exit code 1.
```

After editing the body of the responsible PR, the error disappears.

--- a/src/TagBot/TagBot.jl
+++ b/src/TagBot/TagBot.jl
@@ -45,6 +45,9 @@ function main()
 end
 
 function repo_and_version_of_pull_request_body(body)
+    # Return immediately if the pull request's description is empty. In this
+    # case no further information can be obtained about a registration
+    isnothing(body) && return (nothing, nothing)
     if occursin("JLL package", body)
         @info "Skipping JLL package registration"
         return nothing, nothing

--- a/test/tagbot-unit.jl
+++ b/test/tagbot-unit.jl
@@ -36,6 +36,8 @@ end
     @test TB.repo_and_version_of_pull_request_body(ssh) == ("Foo/Bar", "v1.2.3")
     gitlab = body("https://gitlab.com/Foo/Bar")
     @test TB.repo_and_version_of_pull_request_body(gitlab) == (nothing, "v1.2.3")
+    no_body = nothing
+    @test TB.repo_and_version_of_pull_request_body(no_body) == (nothing, nothing)
 end
 
 @testset "tagbot_file" begin

